repo_e/sum_od_even.py

```diff
@@ -0,0 +1,9 @@
+def sum_of_even(integer):
```

Comment:
nice 👍 please try to implement it using list comprehension and built-in `sum` function

Reply:
I made this